adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+
 def draw_letters():
-    pass
+
+	""" 
+	This function takes no parameters and returns a pool of 10 random letters, 
+	which represents the player's hand. 
+	"""
+	
+	# Initialize available_pool list
+	available_pool = []	
+	
+	# Create dictionary of letter quantities
+	letter_quantities = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 
+    'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 
+    'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+	}
+
+	# Add letters to pool based on quantity in dictionary
+	for letter in letter_quantities:
+		for i in range(letter_quantities[letter]):
+			available_pool.append(letter)
+
+	# Sample 10 random letters from pool and return as list
+	return random.sample(available_pool, 10)
 
 def uses_available_letters(word, letter_bank):
-    pass
+
+	"""
+	This function takes two parameters, a string 'word' and a list of letters 'letter_bank'.
+	Returns True if every letter in 'word' is available in 'letter_bank', otherwise returns False.
+	"""
+
+	# Make copy of letter_bank so that original list is not modified
+	letter_bank_copy = list(letter_bank)
```

Comment:
Would you recommend using the .copy() method as the best way to create a copy? I'm never quite sure since there's a few ways (slice operator, list(), copy())

Reply:
My preference is to use `copy` for clarity of purpose since I believe they all have the same time complexity and slicing and `list` can be used for other things.